src/Symfony/Component/Validator/Constraint.php

```diff
@@ -202,8 +202,6 @@ public function __set(string $option, mixed $value)
      * @return mixed The value of the option
      *
      * @throws InvalidOptionsException If an invalid option name is given
-     *
-     * @internal this method should not be used or overwritten in userland code
```

Comment:
why removing the `@internal` ?

Reply:
The `Valid` and `File` constraints extend this method without making it `@internal`.
To me, this is the sign that extending this method is useful and that it has no reasons to be internal.

Reply:
but why is this hidden in a PR and commit about improving the DebugClassLoader then ? If that's the reason, it deserves to appear on its own.

Reply:
Moved to https://github.com/symfony/symfony/pull/42204